models/Item.go

```diff
@@ -0,0 +1,8 @@
+package models
+
+// Item : Represent json model Item
+type Item struct {
+	Mode                    string  `json:"mode"`
+	StartingCellCoordonates string  `json:"startingCellCoordonates,omitempty"`
```

Comment:
Choose a JSON key format, all in `camelCase` or all in `snake_case` not both, you already used snake case before